tokens/platform-android.json

```diff
@@ -1 +1,338 @@
-{}
\ No newline at end of file
+{
+  "font": {
+    "family": {
+      "sans": {
+        "value": "Roboto",
+        "type": "fontFamilies"
+      },
+      "mono": {
+        "value": "Roboto Mono",
+        "type": "fontFamilies"
+      }
+    },
+    "weight": {
+      "regular": {
+        "value": "400",
+        "type": "fontWeights"
+      },
+      "medium": {
+        "value": "500",
+        "type": "fontWeights"
+      },
+      "bold": {
+        "value": "700",
+        "type": "fontWeights"
+      }
+    },
+    "line-height": {
+      "body": {
+        "xs": {
+          "regular": {
+            "value": "15",
+            "type": "lineHeights"
+          }
+        },
+        "sm": {
+          "regular": {
+            "value": "17",
+            "type": "lineHeights"
+          }
+        },
+        "md": {
+          "regular": {
+            "value": "20",
+            "type": "lineHeights"
+          }
+        },
+        "lg": {
+          "regular": {
+            "value": "22",
+            "type": "lineHeights"
+          }
+        }
+      },
+      "heading": {
+        "sm": {
+          "regular": {
+            "value": "25",
+            "type": "lineHeights"
+          }
+        },
+        "md": {
+          "regular": {
+            "value": "27",
+            "type": "lineHeights"
+          }
+        },
+        "lg": {
+          "regular": {
+            "value": "34",
+            "type": "lineHeights"
+          }
+        },
+        "xl": {
+          "regular": {
+            "value": "41",
+            "type": "lineHeights"
+          }
+        }
+      }
+    },
+    "size": {
+      "body": {
+        "xs": {
+          "value": "11",
+          "type": "fontSizes"
+        },
+        "sm": {
+          "value": "12",
+          "type": "fontSizes"
+        },
+        "md": {
+          "value": "14",
+          "type": "fontSizes"
+        },
+        "lg": {
+          "value": "16",
+          "type": "fontSizes"
+        }
+      },
+      "heading": {
+        "sm": {
+          "value": "20",
+          "type": "fontSizes"
+        },
+        "md": {
+          "value": "22",
+          "type": "fontSizes"
+        },
+        "lg": {
+          "value": "28",
+          "type": "fontSizes"
+        },
+        "xl": {
+          "value": "34",
+          "type": "fontSizes"
+        }
+      }
+    },
+    "letter-spacing": {
+      "body": {
+        "xs": {
+          "value": "0.5px",
+          "type": "letterSpacing"
+        },
+        "sm": {
+          "value": "0.4px",
+          "type": "letterSpacing"
+        },
+        "md": {
+          "value": "0.25px",
+          "type": "letterSpacing"
+        },
+        "lg": {
+          "value": "0.25px",
+          "type": "letterSpacing"
+        }
```

Comment:
Both iOS and Web use percentages for letter spacing. Is that expected to have pixel values here?

And for my own understanding, what's the rationale behind that?

Reply:
I added the rationale to the internal doc on transforming typography tokens, but just realized I haven't pushed it yet. Let me do it now.

Reply:
Actually, I pushed it yesterday. https://github.com/vector-im/compound-internal/blob/internal-docs/docs/typography.md#android